### PR TITLE
ykw

### PR DESCRIPTION
**Pull request by rujiangfanfanzi**
ykw 作业提交

### CODE REVIEW

File: ykw/2017-11-14/hanoi/han.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = function hanoi(n,x,y,z) {
```
Comment (calidion) [inline ykw/2017-11-14/hanoi/han.js]: module.exports = function hanoi(n,x,y,z) {
=>
module.exports = function hanoi(n,x,y,z) {

File: ykw/2017-11-14/hanoi/han.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    if(n==1){
+        console.log("Move "+n+" from "+x+" to "+z);
+    }
+    else{
+        hanoi(n-1,x,z,y);
```
Comment (calidion) [inline ykw/2017-11-14/hanoi/han.js]: hanoi(n-1,x,z,y);
=》
 hanoi(n-1, x, z, y);

File: ykw/2017-11-14/hanoi/han.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = function hanoi(n,x,y,z) {
+    if(n==1){
+        console.log("Move "+n+" from "+x+" to "+z);
+    }
+    else{
```
Comment (calidion) [inline ykw/2017-11-14/hanoi/han.js]: }
else{ 
=>
} else {

File: ykw/2017-11-14/hanoi/han.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+module.exports = function hanoi(n,x,y,z) {
+    if(n==1){
```
Comment (calidion) [inline ykw/2017-11-14/hanoi/han.js]: if (n==1){
=>
if (n == 1) {

File: ykw/2017-11-14/hanoi/han2.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+var han = require('./han.js');
+var arg = process.argv.splice(2);
+hanoi(arg[0],arg[1],arg[2],arg[3]);
```
Comment (calidion) [inline ykw/2017-11-14/hanoi/han2.js]: hanoi(arg[0],arg[1],arg[2],arg[3]);
=>
hanoi(arg[0], arg[1], arg[2], arg[3]);

**Comment by rujiangfanfanzi** [comment]
ykw 格式修改

**Pull request merged**